uwblueprint/website-bp
# [INTF23] Updates search, filtering, and sorting functionality

### Implementation description

- working search, filtering, and sorting functionality



### Steps to test

1. Try searching for names, status, etc
2. Try toggling some filters to see how dashboard is updated
3. Try sorting the columns and see if expected matches actual behavior 




### What should reviewers focus on?

-

### Checklist
